.travis.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         - pip install -r conf/requirements.txt
       script:
         - "flake8 $(file generators/* tools/* | sed -ne 's/:.*python.*//pI')"
+        - "flake8 $(file tools/*.py tools/runners/*.py)"
```

Comment:
I think we cannot do it like this after all. It needs to be a part of the same command, otherwise if the first check fails, the second is not run.

Reply:
`flake8` now checks all files in a single command